scripts/generate_tilesets.py

```diff
@@ -806,6 +780,8 @@ def main(cfg_file_path):
     img_md_df.rename(columns={"index": "img_file"}, inplace=True)
 
     img_md_df['id'] = img_md_df.apply(misc.img_md_record_to_tile_id, axis=1)
+    if img_md_df.year.isna().all():
```

Comment:
Those lines provoke an error when the WMS or MIL connectors are selected. I rather propose to modify the way the FOLDER metadate is built (see l107 FOLDER.py) and delete these lines.